### DISCUSSION TITLE
Does rdkafka support dynamic topics in regex based subscription

### DISCUSSION THREAD

**Discussion by atul-raghuwanshi** [edenhill/librdkafka#4797] [General]
i am using rdkafka 1.9.0. When i am using regex based subscription it is only matching and subscribing using regex during subscribe. No new topics matching the regex are getting assigned dynamically.

**Comment by atul-raghuwanshi** [comment]
@edenhill can you tell me the info as i am not able to find it anywhere in documentations.

**Reply by mensfeld** [reply]
Not true. They are getting discovered later based on your metadata refresh settings.

**Reply by atul-raghuwanshi** [reply]
Ah yes, i found the intended behavior in a older conversation. It waits for the metadata refresh like java so 5 minutes by default. Although the documentation for this is hard to find. 

Verified that it works correctly.

**Comment by atul-raghuwanshi** [comment]
Closing this as nothing is pending.
